query-engine/driver-adapters/Cargo.toml

```diff
@@ -22,9 +21,21 @@ bigdecimal = "0.3.0"
 chrono = "0.4.20"
 futures = "0.3"
 
-napi.workspace = true
-napi-derive.workspace = true
-
 [dev-dependencies]
 expect-test = "1"
 tokio.workspace = true
+
+[target.'cfg(not(target_arch = "wasm32"))'.dependencies]
+napi.workspace = true
+napi-derive.workspace = true
+quaint.workspace = true
+
+[target.'cfg(target_arch = "wasm32")'.dependencies]
+quaint = { path = "../../quaint" }
+js-sys.workspace = true
+serde-wasm-bindgen.workspace = true
+wasm-bindgen.workspace = true
+wasm-bindgen-futures.workspace = true
+tsify.workspace = true
+ducktor = "0.1.0"
+pin-project = "1"
```

Comment:
I believe you should be able to use quaint's workspace version in both cases here. [See doc here](https://doc.rust-lang.org/cargo/reference/resolver.html#feature-resolver-version-2)

<img width="797" alt="image" src="https://github.com/prisma/prisma-engines/assets/25233379/045a20ad-0ca9-461d-a156-65df32a9f00f">

Reply:
Thanks, I didn't know this about `resolver = "2"`.